x-pack/plugins/enterprise_search/public/applications/shared/kibana/kibana_logic.ts

```diff
@@ -22,6 +22,7 @@ import {
 import { DataPublicPluginStart } from '@kbn/data-plugin/public';
 
 import { GuidedOnboardingPluginStart } from '@kbn/guided-onboarding-plugin/public';
+import { IndexMappingProps } from '@kbn/index-management-plugin/public/application/sections/home/index_list/details_page/index_mapping_with_context_types';
```

Comment:
```suggestion
import type { IndexMappingProps } from '@kbn/index-management-plugin/public/application/sections/home/index_list/details_page/index_mapping_with_context_types';
```
nit: generally speaking, types like these should be exported from `public/index.ts`, especially across plugins.